src/modules/core/components/Icon/icons.json

```diff
@@ -46,6 +46,7 @@
     "tag",
     "three-dots-row",
     "unlock",
+    "upvote",
```

Comment:
This is a vertically flipped version of our `caret-down-small`:

![Screenshot from 2020-01-22 15-22-52](https://user-images.githubusercontent.com/1193222/72897675-2a40cf80-3d2b-11ea-8fa4-f62650c5828b.png)

So by my reasoning this should be called `care-up-small`

Reply:
Done! Great call 🍻